data-types.md

```diff
@@ -199,9 +195,13 @@ A Trip is defined by the following structure:
 | Field                    | Type            | Required/Optional      | Comments |
 | -----                    | ----            | -----------------      | -------- |
 | `provider_id`            | UUID            | Required               | A UUID for the Provider, unique within MDS. See MDS [provider list](/providers.csv). |
+| `provider_name` | String | Required | The public-facing name of the Provider |
```

Comment:
I always hated having the `provider_name` as well as the `provider_id` because of the redundancy.  I will defer to the will of the group but I want to make my case that it's a stupid-simple lookup on a small table to get the provider name.

Reply:
Yes agreed, my mistake. We have talked about this in WG meetings and were waiting for the next major breaking release.